ProMotion.gemspec

```diff
@@ -25,6 +25,9 @@ Gem::Specification.new do |gem|
   gem.version       = ProMotion::VERSION
 
   gem.add_runtime_dependency("methadone", "~> 1.7")
+  gem.add_runtime_dependency("motion_print", "~> 1.0")
+
+  gem.add_development_dependency("motion_print", "~> 1.0")
```

Comment:
Cool. Thought you said that in the original issue. Could be remembering wrong too. I'll update this tomorrow morning!